### ISSUE TITLE
Feature : nested albums

### ISSUE THREAD

**Issue by tomchiverton** [pulsejet/memories#523]
**Is your feature request related to a problem? Please describe.**
I have ~20 years of photos. I have several holidays each year. I want to put all my holidays in a "Holidays" album, with sub albums for each holiday. I do not want to change from storing (a single copy) of each photo, my year and month, as uploaded from my phone with the Nextcloud app

**Describe the solution you'd like**
In "add album", allow choosing a parent album.

**Describe alternatives you've considered**
Creating and managing a parallel set of folders

**Comment by luxzg** [comment]
I can relate to that. My example is a big festival, which has several classes a day, and a party each night. Being able to have album "Festival 2023" and beneath it sub-albums like "Party day 1", "Party day 2", "Class 1", "Class 2", "Workshop 3" would be great. It's not extremely high on my own priorities list, as I'm still in process of sorting thousands of photos in thw first place, but it would be great to create parent albums when I'm done, and attach existing albums to parents.

**Comment by theST4GZ** [comment]
This is the only option stopping me from using Memories full-time. Memories is the best way for me and my wife to share our albums but without sub-albums there are just too many albums to filter.

**Comment by hbarel** [comment]
Agreed. It seems as the team puts a lot of efforts into scalability, but I cannot imagine a collection of more than 150-200 photos using a flat list of albums.

**Comment by smokinjo** [comment]
I am also interested in any update on this?

It would allow having a hierarchy to manage the albums better.

Thanks

**Comment by nick-niccoo** [comment]
Also interested in this feature! Hope this will become available any time soon.

**Comment by MrRinkana** [comment]
Cant you just structure folders as proposed and put the images there, ie:
Holidays/
├─ new year 2024/
├─ autumn break 1988/
├─ etcetera..

then navigate to the holidays directory in the folder tab (within memories) and there change to timeline view? It will have the same effect, just it not being an "album".

In other words organising in folder-based albums?

If one wants to use albums in the above context then #810 might be of interest

**Comment by gadolf66** [comment]
Hi! same here, would be a great feature, coupled with #810 and without loosing the flexibility of loosing the flexibility of adding photos to Albums from multiple locations as it is in its current state, as stated [here](https://github.com/pulsejet/memories/issues/810#issuecomment-2016300473).

**Comment by pnyccz** [comment]
I'm also really interested in this feature. Is there anything we can do to help get this started?

**Comment by sbglt** [comment]
I wrote #1440 before I found this request. Digikam is my tagging tool and I want to share some of them in Nextcloud.
I have lots of hierarchical tags (inherited from lightroom).

I wrote a python script to synchronise digikam with Nextcloud. I should use API but I didn't found what I was looking for. So the script reads the digikam database and writes to nextcloud's one. It's ugly but this first revision works for my use case. 

I'm a C/C++ developper, but not a web developper. I could try to participate with help.
